wagn/lib/wagn/mods_spec_helper.rb

```diff
@@ -1,6 +1,5 @@
 # -*- encoding : utf-8 -*-
 require 'spork'
-raise "is this loaded anywhere?"
```

Comment:
> raise "is this loaded anywhere?"
yes, this is loaded in the spec_helper.rb file that is generated for mod-dev decks (for example like wikirate - this was the reason why the tests on semaphore failed for wikirate)